UT4MasterServer/Services/StatisticsService.cs

```diff
@@ -0,0 +1,2203 @@
+﻿using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+using UT4MasterServer.DTOs;
+using UT4MasterServer.Enums;
+using UT4MasterServer.Models;
+using UT4MasterServer.Other;
+
+namespace UT4MasterServer.Services;
+
+public sealed class StatisticsService
+{
+	private readonly ILogger<StatisticsService> logger;
+	private readonly IMongoCollection<Statistic> statisticsCollection;
+	private readonly bool allowPasswordGrant;
+
+	public StatisticsService(
+		ILogger<StatisticsService> logger,
+		DatabaseContext dbContext,
+		IOptions<DatabaseSettings> settings)
+	{
+		this.logger = logger;
+		statisticsCollection = dbContext.Database.GetCollection<Statistic>("statistics");
+		allowPasswordGrant = settings.Value.AllowPasswordGrantType;
+	}
+
+	public async Task<List<StatisticDto>> GetDailyAccountStatistics(string accountId)
+	{
```

Comment:
just like everywhere else `accountId` should be of type `EpicID`.

im not sure if this method is used, but two of them below look like copy-paste. cant they my merged into one?
`GetAggregateAccountStatistics` already seems like it can be used for any windows of statistics, no?

Reply:
The methods were different, but I changed something. I will merge them into one.